src/modules/mavlink/streams/RAW_RPM.hpp

```diff
@@ -49,29 +49,33 @@ class MavlinkStreamRawRpm : public MavlinkStream
 
 	unsigned get_size() override
 	{
-		return _rpm_sub.advertised() ? (MAVLINK_MSG_ID_RAW_RPM_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES) : 0;
+		return _rpm_subs.advertised() ? (MAVLINK_MSG_ID_RAW_RPM_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES) : 0;
```

Comment:
Technically to not screw up the size and throttling calculation you should return the total size.

```suggestion
		return _rpm_subs.advertised_count() * (MAVLINK_MSG_ID_RAW_RPM_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES);
```